### PR TITLE
added elasticsearch to docs readme

### PR DESCRIPTION
**Pull request by puja108**
RFR @giantswarm/cena

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -9,4 +9,5 @@ Here we document important concepts and functionality Inago supports.
 - [Unit file structure](structure.md)
 - [Terminology](terminology.md)
 - [Deploy Kubernetes on CoreOS with Inago](k8s.md)
+- [Deploy Elasticsearch with Inago](elasticsearch.md)
```
Comment (JosephSalisbury) [inline docs/README.md:12]: Hmm, I don't like that this isn't consistent with the Kubernetes link.

Comment (puja108) [inline docs/README.md:12]: I'd remove the CoreOS thingy, what do you think?

Comment (JosephSalisbury) [inline docs/README.md:12]: From the Kubernetes link? That sounds good to me.

**Comment by puja108** [comment]
@JosephSalisbury removed, LGTY?

**Comment by JosephSalisbury** [comment]
LGTM

**Pull request merged**